cbox/cliparser.py

```diff
@@ -107,4 +124,13 @@ def _param2args(param, doc_param=None):
     else:
         arg_type = str
 
-    return arg_name, arg_type, arg_default, arg_required, arg_help
+    if not isinstance(arg_type, Arg):
+        arg_type = Arg(type_=arg_type)
+
+    if arg_type.names is None:
+        arg_type.names = [param.name.replace('_', '-')]
```

Comment:
Maybe add both options? (`-` and `_`)?

    arg_type.names = [param.name, param.name.replace('_', '-')]